recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -70,8 +69,12 @@ RESOURCE_CPU_WEIGHT = "100"
 RESOURCE_MEMORY_HIGH = "100M"
 RESOURCE_MEMORY_MAX = "80%"
 
-do_compile_ptest() {
-    cmake_runcmake_build --target build_tests "${PARALLEL_MAKE}"
+do_configure_prepend() {
+    # CMake has trouble finding yocto's git when cross-compiling, let's do this step manually
+    cd ${S}
+    if [ ! -f VERSION ]; then
+        ./scripts/get_version.sh > VERSION
```

Comment:
Is this relying on git being installed on the host? Seems okay, though.

Reply:
git is listed in yocto's dependencies and is used to fetch aktualizr as well.